app/Eloquent/Book.php

```diff
@@ -65,7 +65,12 @@ public function updateBooks()
 
     public function medias()
     {
-        return $this->morphMany(Media::class, 'target')->orderBy('priority', 'desc');
+        return $this->morphMany(Media::class, 'target')
+            ->orderBy('priority', 'desc')
+            ->select([
+                'target_id',
+                'path'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Eloquent/Book.php</br>Comma required after last value in array declaration